Tapawingo_backend/Services/EditionsService.cs

```diff
@@ -0,0 +1,52 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using Tapawingo_backend.Dtos;
+using Tapawingo_backend.Interface;
+using Tapawingo_backend.Models;
+
+namespace Tapawingo_backend.Services
+{
+    public class EditionsService
+    {
+        private readonly IMapper _mapper;
+        private readonly IEditionsRepository _editionsRepository;
+        private readonly IOrganisationsRepository _organisationsRepository;
+        private readonly IEventsRepository _eventsRepository;
+
+        public EditionsService(IMapper mapper, IEditionsRepository editionRepository, IOrganisationsRepository organisationsRepository, IEventsRepository eventsRepository) {
+            _editionsRepository = editionRepository;
+            _mapper = mapper;
+            _organisationsRepository = organisationsRepository;
+            _eventsRepository = eventsRepository;
+        }
+
+        public Edition CreateEdition(CreateEditionDto model, int organisationId, int eventId)
+        {
+            if (!_organisationsRepository.OrganisationExists(organisationId))
+            {
+                throw new ArgumentException("Organisation does not exist");
```

Comment:
Het is beter om geen exceptions the throwen dit is een vrij dure operatie. IActionResults returnen is de betere manier van foutafhandeling